src/component/Books.js

```diff
@@ -14,7 +15,7 @@ const Books = () => {
   };
 
   return (
-    <div className="books">
+    <div className={styles.books}>
       <BookList books={books} onDelete={handleDeleteBook} />
       <BookForm onAdd={handleAddBook} />
```

Comment:
- [ ] @vanina123 great work on your books styling , however, the books take all the weight instead of leaving them a margin, also each book should be wrapped in a container and their width should match that width of the form, check the comparison below

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/vanina123/books-store/assets/61408860/70761177-0bdf-4666-abaf-bef7709bf0ba) | ![image](https://github.com/vanina123/books-store/assets/61408860/7f9709cf-3f28-400e-85a1-5ee05e5d8122)